### PR TITLE
[MIG] stock_orderpoint_generator: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: stock_orderpoint_generator/views/orderpoint_template_views.xml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    <record model="ir.ui.view" id="view_warehouse_orderpoint_template_search">
+        <field name="name">stock.warehouse.orderpoint.template.search</field>
+        <field name="model">stock.warehouse.orderpoint.template</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <search string="Reordering Rule Templates Search">
+                <field name="name" string="Reordering Rule Templates"/>
+                <field name="warehouse_id"/>
+                <field name="location_id" groups="stock.group_locations"/>
```
Comment (chienandalu) [inline stock_orderpoint_generator/views/orderpoint_template_views.xml]: Shouldn't it be `stock.group_stock_multi_locations`

File: stock_orderpoint_generator/views/orderpoint_template_views.xml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                            attrs="{'invisible': [('auto_generate', '=', False)]}"/>
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                        </group>
+                        <group>
+                            <field name="warehouse_id" on_change="onchange_warehouse_id(warehouse_id)" widget="selection" groups="stock.group_locations"/>
+                            <field name="location_id" groups="stock.group_locations"/>
```
Comment (chienandalu) [inline stock_orderpoint_generator/views/orderpoint_template_views.xml]: Same here with `stock.group_stock_multi_locations`?

File: stock_orderpoint_generator/views/orderpoint_template_views.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+<odoo>
+
+    <record id="view_warehouse_orderpoint_template_tree" model="ir.ui.view">
+        <field name="name">stock.warehouse.orderpoint.template.tree</field>
+        <field name="model">stock.warehouse.orderpoint.template</field>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <tree string="Reordering Rule Templates">
+                <field name="name"/>
+                <field name="warehouse_id" groups="stock.group_locations"/>
```
Comment (chienandalu) [inline stock_orderpoint_generator/views/orderpoint_template_views.xml]: `stock.group_stock_multi_locations`

File: stock_orderpoint_generator/views/orderpoint_template_views.xml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+                <field name="location_id" groups="stock.group_locations"/>
```
Comment (chienandalu) [inline stock_orderpoint_generator/views/orderpoint_template_views.xml]: `stock.group_stock_multi_locations`

File: stock_orderpoint_generator/views/orderpoint_template_views.xml [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+                            help="Reordering rules will be created for the selected products. This is equivalent to the Scheduled Action action."
+                            attrs="{'invisible': [('auto_generate', '=', False)]}"/>
+                </header>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="name" />
+                        </group>
+                        <group>
+                            <field name="warehouse_id" on_change="onchange_warehouse_id(warehouse_id)" widget="selection" groups="stock.group_locations"/>
```
Comment (chienandalu) [inline stock_orderpoint_generator/views/orderpoint_template_views.xml]: is this `onchange` attribute necessary anymore?

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Comment by cubells** [comment]
@chienandalu done.

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@chienandalu Can you update your review ?

**Review by chienandalu — APPROVED** [review]
Great :+1:

**Pull request merged**